resources/views/service/index_registration.blade.php

```diff
@@ -37,7 +37,9 @@
                             <td class="center">{{ $registration->family->entitlement }}</td>
                             <td class="center">{{ $registration->family->rvid }}</td>
                             <td>
-                            <button onclick="window.location.href='{{ URL::route('service.registration.edit', ['id' => $registration->id ]) }}'">Select</button>
+                            <button onclick="window.location.href='{{ URL::route('service.registration.edit', ['id' => $registration->id ]) }}'"
+                                @if( isset($registration->family->leaving_on) ) disabled @endif
+                            > Select</button>
```

Comment:
How would you turn them back on? Or is that not something we'd ever want to need?

Reply:
When re-enable people becomes a thing, then we'll do that. At the moment they'll probably just reregister them as a second family record.

Reply:
Fair.